frontend/src/components/managePermissions/table/UserDataTable.vue

```diff
@@ -33,19 +33,10 @@ const props = defineProps({
         >,
         required: true,
     },
-    selectedApplicationDisplayText: {
-        type: String,
-        requried: true,
-    },
-    isApplicationSelected: {
-        type: Boolean,
-        required: true,
-        default: false,
-    },
 });
 
-const filters = ref({
-    global: { value: null, matchMode: FilterMatchMode.CONTAINS },
+const userAssignmentsfilters = ref({
```

Comment:
userAssignmentsFilters?

Reply:
and you might need to add the user type in the filter, cause below you're using it
<img width="863" alt="Screen Shot 2024-01-15 at 5 56 18 PM" src="https://github.com/bcgov/nr-forests-access-management/assets/23131735/aaa729c1-604c-4ea1-bc45-df956c83e27a">

Reply:
added the missing filter, is the naming userRoleAssignmentsFilters ok?